tests/phpunit/integration/Modules/Tag_ManagerTest.php

```diff
@@ -360,6 +361,76 @@ public function test_get_datapoints() {
 		);
 	}
 
+	public function test_get_assets() {
+		$context    = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
+		$tagmanager = new Tag_Manager( $context );
+
+		$assets = $tagmanager->get_assets();
+
+		$this->assertCount( 1, $assets );
+
+		$script = $assets[0];
+		$script->register( $context );
+
+		$dependency = wp_scripts()->registered['googlesitekit-modules-tagmanager'];
+
+		$this->assertEquals( $context->url( 'dist/assets/' ) . 'js/googlesitekit-modules-tagmanager.js', $dependency->src );
+		$this->assertEqualSets(
+			array(
+				'googlesitekit-api',
+				'googlesitekit-data',
+				'googlesitekit-datastore-site',
+				'googlesitekit-modules',
+				'googlesitekit-vendor',
+				'googlesitekit-modules-analytics',
+			),
+			$dependency->deps
+		);
+
+		wp_deregister_script( 'googlesitekit-modules-tagmanager' );
```

Comment:
Is this necessary? I would expect this to not be persisted between tests . If it does need to be cleaned up, let's do that in a before/after method to ensure it's cleaned up properly rather than in the test where it won't run if the test fails which could cause other tests to fail as a result.

Reply:
It is necessary, unfortunately. Good shout though, I've moved the `wp_deregister_script` calls to to a `tear_down` method.